src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -1405,29 +1492,112 @@ const cleanupSubscriptions = () => {
+
+Because the schema has a one to many relationship enabled between posts and comments, you can use the auto-generated field `postCommentsId` that defines the relationship between the post and the comment to set this up in a new Subscription definition.
+
+To implement this, you could update the schema with the following:
+
+```graphql
+type Post @model @auth(rules: [{ allow: public, provider: apiKey }]){
+  id: ID!
+  title: String!
+  content: String
+  comments: [Comment] @hasMany
+}
+
+type Comment @model @auth(rules: [{ allow: public, provider: apiKey }]){
+  id: ID!
+  content: String
+  postCommentsId: ID!
+}
+
+type Subscription {
+  onCommentByPostId(postCommentsId: ID!): Comment
+    @aws_subscribe(mutations: ["createComment"])
+}
+
+```
 
-[comment]: # (Explanation of how to unsubscribe and when to for reducing redundancy with steps for these actions.)
+Now you can create a custom subscription for comment creation with a specific post id:
 
-You can unsubscribe from events by using subscriptions, a GraphQL feature that allows the server to send data to its clients when a specific event happens. You can enable real-time data integration in your app with a subscription.
-[question]: (RENE, changes OK in line 1413?)
+```js
+import { API } from 'aws-amplify';
+import { onCommentByPostId } from './graphql/subscriptions';
+
+API.graphql({
+  query: onCommentByPostId,
+  variables: {
+    postCommentsId: "12345"
+  }
+})
+.subscribe({
+  next: data => {
+    console.log('data: ', data)
+  }
+})
+```
+
+</Accordion>
+
+### Unsubscribe from a subscription
+
+You can also unsubscribe from events by using subscriptions by implementing the following:
 
 ```ts
 // Stop receiving data updates from the subscription
 sub.unsubscribe();
 ```
 
-### Recap
-You now have set up subscriptions for real-time events and understand how to filter and cancel these subscriptions when needed.
-
-[comment]: # (ADD TO RECAP a tie-in to real-world use and application of the topic discussed OR reiterate the benefit.)
+**Recap:** You now have set up subscriptions for real-time events and understand how to filter and cancel these subscriptions when needed.
 
 ## Conclusion
 
 Congratulations! You have finished the **Connect a data stack** guide. In this guide, you installed and configured Amplify Libraries; used GraphQL mutations for app data requests; filtered app data in Amplify; and set up, filtered, and canceled subscriptions for real-time events.
 
 ## Next steps
 
-[comment]: # (Provide 2–3 relevant next steps. These can be links related to what was covered in the guide and/or links to related Amplify Job to be done guides.)
-[comment]: # (RENE, next steps are missing.)
\ No newline at end of file
+Our recommended next steps include continuing to build out and customize your information architecture for your data. Some resources that will help with this work include:
```

Comment:
I don't see it on other pages but do we want to add 1-sentence descriptions on what these pages entail? They are also all CLI pages - any specific heads-up there?

Reply:
IMHO, we could use Claude to summarize a one-sentence description. See if that lines up with our expectations. It'd be a good test to see if our content actually meets what we set them out to do as well? Otherwise I think the fastest way to pull one-sentence descriptions would be from the `const meta = { ... }` of those pages.